netebpfext/net_ebpf_ext.h

```diff
@@ -49,6 +49,9 @@
 #define NET_EBPF_EXTENSION_POOL_TAG 'Nfbe'
 #define NET_EBPF_EXTENSION_NPI_PROVIDER_VERSION 0
 
+#define htonl(x) _byteswap_ulong(x)
+#define htons(x) _byteswap_ushort(x)
```

Comment:
https://github.com/microsoft/ebpf-for-windows/pull/916/files#diff-6e4bda166b4c5a3103506c3378f2cfa27ee4cc3edfba5ef7dc01ec7ecfc943f1 adds bpf_endian.h, maybe we can combine this into there?

Reply:
I have added bpf_htonl etc. in the new `bpf_endian.h` file. Those maybe used by bpf programs. But as far as the netebpfext.sys driver is concerned, since it targets windows only, IMO its OK to use DDK functions such as rtlswapbytes etc.